components/locale/src/fallback/algorithms.rs

```diff
@@ -256,54 +356,79 @@ mod tests {
                 "sr-Latn-fonipa",
                 "sr-Latn",
             ],
+            expected_script_chain: &[
+                "sr-ME-fonipa",
+                "sr-ME",
+                "sr-fonipa",
+                "sr",
+                "und-Latn-fonipa",
+                "und-Latn",
+                "und-fonipa",
+            ],
             expected_region_chain: &["sr-ME-fonipa", "sr-ME", "und-ME-fonipa", "und-ME"],
         },
         TestCase {
             input: "sr-RS",
             requires_data: true,
             expected_language_chain: &["sr-RS", "sr"],
+            expected_script_chain: &["sr-RS", "sr", "und-Cyrl"],
```

Comment:
Question, here and elsewhere: should `und-Cyrl-RS` be included? I'm okay if not.

Reply:
Discussion: there will not be any data here, so skip it